pkg/mcs/scheduling/server/server.go

```diff
@@ -212,6 +212,11 @@ func (s *Server) updateAPIServerMemberLoop() {
 				cc, err := s.GetDelegateClient(ctx, s.GetTLSConfig(), ep.ClientURLs[0])
 				if err != nil {
 					log.Info("failed to get delegate client", errs.ZapError(err))
+					continue
+				}
+				if !s.IsServing() {
+					// double check
+					break
```

Comment:
Do we need to add a `break label` to indicate jumping out of the “for loop”?

Reply:
I think it is not necessary.